uniffi_bindgen/src/templates/ErrorTemplate.rs

```diff
@@ -12,7 +12,7 @@ impl From<{{e.name()}}> for uniffi::deps::ffi_support::ExternError {
         // we might find that we need to reserve some codes.
         match err {
             {%- for value in e.values() %}
-            {{ e.name()}}::{{value}} => uniffi::deps::ffi_support::ExternError::new_error(uniffi::deps::ffi_support::ErrorCode::new({{ loop.index }}), err.to_string()),
+            {{ e.name()}}::{{value}}(..) => uniffi::deps::ffi_support::ExternError::new_error(uniffi::deps::ffi_support::ErrorCode::new({{ loop.index }}), err.to_string()),
```

Comment:
If you change the `(..)` to `{ .. }`, I think you can support both plain and wrapped variants! 😄🎉

Reply:
I swear my jaw just dropped. This is exactly what I was looking for 🎉 😄 🚀 🕺

Reply:
Boom! Nice one @linacambridge 😁 

(I don't have a good mental model of *why* that syntax works, because in my head `{}` is for struct literals so I didn't expect it to work with the tuple-like variant..but I love it!)